src/components/pages/AdminHome/AdminCourses.js

```diff
@@ -1,11 +1,80 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import AdminSidebar from './AdminSidebar';
 import '../../../styles/index.less';
```

Comment:
`index.less` shouldn't need to be directly imported into a file. It should added to the root directly. If it is not working, we can make a bug ticket.